ament_clang_tidy/ament_clang_tidy/main.py

```diff
@@ -66,6 +66,15 @@ def main(argv=sys.argv[1:]):
         '--fix-errors',
         action='store_true',
         help='Fix the suggested changes')
+    parser.add_argument(
+        '--quiet',
+        action='store_true',
+        help='Suppresses printing statistics about ignored warnings '
+             'and warnings treated as errors')
+    parser.add_argument(
+        '--add-headers',
```

Comment:
Is there anything wrong with exposing the option as-is from clang-tidy? I.e. change this to `--header-filter` and accept a regex for header files.
If we do this, make sure to update the "help" description.

Reply:
Removed the binary option of specifying non-system header output and added support for ``-header-filter`` supplied with regex